README.md

```diff
@@ -54,7 +54,9 @@ After that the documentation will be available in `target/docs/mod-users-keycloa
 | KC_LOGIN_CLIENT_SUFFIX           | -login-application         |  false   | Suffix of a Keycloak client who owns the authorization resources. It is used as `audience` for keycloak when evaluating permissions. |
 | MIGRATION_BATCH_SIZE             | 20                         |  false   | Batch size for user migration. Max value is 50                                                                                       |
 | DEFAULT_PASSWORDS_ON_MIGRATION   | false                      |  false   | If specified to true migrated user’s password being set to their username otherwise migrated users not having any credentials set    |
-| INCLUDE_ONLY_VISIBLE_PERMISSIONS | true                       |  false   | Defines if onlyVisible (UI permisisons/permission-set names) will be returned using `_self` endpoint                                 |
+| INCLUDE_ONLY_VISIBLE_PERMISSIONS | true                       |  false   | Defines if onlyVisible (UI permissions/permission-set names) will be returned using `_self` endpoint                                 |
+| SINGLE_TENANT_UX                 | false                      |  false   | Defines if the module is running in single tenant UX mode                                                                            |
+| IDENTITY_PROVIDER_SUFFIX         | {tenantId}-keycloak-oidc   |  false   | Suffix of a Keycloak OIDC identity provider who will perform federated authentication requests. Used if SINGLE_TENANT_UX is enabled  |
```

Comment:
let's follow the same convention as for KC_LOGIN_CLIENT_SUFFIX, in any case we should not allow user to override {tenantId} , so result for default value should be like this: "-keycloak-oidc"